src/lib/api/resources/Accounts.ts

```diff
@@ -0,0 +1,30 @@
+import { AxiosInstance } from "axios";
+
+import CONFIG from "../../../config";
+
+import { User } from "../../../types/api/accounts";
+
+const { CLIENT_SETTINGS } = CONFIG;
+const { GENERAL_ENDPOINT } = CLIENT_SETTINGS;
+
+export default class AccountsApi {
+  private client: AxiosInstance;
+
+  private accountsURL: string;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+    this.accountsURL = `${GENERAL_ENDPOINT}/api/accounts`;
+  }
+
+  /**
+   * Get a list of your Mailtrap accounts.
+   * @returns Returns the list of accounts to which the API token has access.
+   * access_levels can return 1000 (account owner), 100 (admin), 10 (viewer).
```

Comment:
these should be constants or an enum?

Reply:
will introduce them in return type instead